src/scalar_mul/tests.rs

```diff
@@ -0,0 +1,82 @@
+use alloc::vec::Vec;
+use group::GroupEncoding;
+use jubjub::{ExtendedPoint, Scalar};
+
+use crate::scalar_mul::VartimeMultiscalarMul;
+
+// /// This function can be used to generate the data for [`test_jubjub_vartime_multiscalar_mul`].
+// #[test]
+// fn gen_jubjub_data() {
+//     use group::{ff::Field, Group};
+//     use rand::thread_rng;
+//     use std::println;
+
+//     let rng = thread_rng();
+
+//     let scalars = [Scalar::random(rng.clone()), Scalar::random(rng.clone())];
+//     println!("Scalars:");
+//     for scalar in scalars {
+//         println!("{:?}", scalar.to_bytes());
+//     }
+
+//     let points = [
+//         ExtendedPoint::random(rng.clone()),
+//         ExtendedPoint::random(rng),
+//     ];
+//     println!("Points:");
+//     for point in points {
+//         println!("{:?}", point.to_bytes());
+//     }
+
+//     let res = ExtendedPoint::vartime_multiscalar_mul(scalars, points);
+//     println!("Result:");
+//     println!("{:?}", res.to_bytes());
+// }
```

Comment:
Same as above

Reply:
Done in [6a048b2](https://github.com/ZcashFoundation/reddsa/pull/57/commits/6a048b28f749263480eaa4d42d3d141c7dcbddaa).